### ISSUE TITLE
npm install of dredd not working

### ISSUE THREAD

**Issue by Simo72** [apiaryio/dredd#921]
I'm trying to install dredd and after following the docs, along with known npm fixes, I get the same warning

Used npm install of node and npm and tried to separately install fury@3.0.0-beta. 4 with no success

Dredd is added to the package-lock.json but no dredd commands (eg. dredd --help), seem to work (the same warning appears installing locally and globally)
```
npm WARN fury-adapter-apib-parser@0.9.0 requires a peer of fury@3.0.0-beta.4 but none is installed. You must install peer dependencies yourself.
npm WARN fury-adapter-swagger@0.13.4 requires a peer of fury@3.0.0-beta.4 but none is installed. You must install peer dependencies yourself.
```
Thanks for any help

**Comment by honzajavorek** [comment]
Hi, this is a duplicate of https://github.com/apiaryio/dredd/issues/905 and I believe it will be resolved soon by https://github.com/apiaryio/dredd/pull/906, which I'm currently working on. Bear with us! 🐻

**Comment by honzajavorek** [comment]
@Simo72 This should be now fixed in [v4.7.1](https://github.com/apiaryio/dredd/releases/tag/v4.7.1). Would you mind to confirm it's okay now?

**Comment by SOPHIA2401** [comment]
Hi @honzajavorek,
I'm currently using `dredd v14.1.0` in my project and facing the above issue.
